### ISSUE TITLE
Logging module

### ISSUE THREAD

**Issue by IceFlovver** [amosproj/amos2022ws01-firmware-scraper#18]
## User story
1. As a user of the software
2. I want it to check if logging in is necessary in order to download firmware and if the logging credentials are already available
3. So that the firmware can be downloaded

## Acceptance criteria
* it is possible to determine whether logging in is necessary for downloading firmware
* the software can check whether credentials for logging in already exist in the database
* the software is able to download the firmware that requires logging in
* in case there are no credentials the software sends a message to the user

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
